src/composables/useGetMessages.ts

```diff
@@ -539,6 +551,66 @@ export function useGetMessagesProvider() {
 		}, 500)
 	}
 
+	/**
+	 * Try to start chat relay
+	 */
+	function tryChatRelay() {
+		if (chatRelaySupported && isChatEndReached.value) {
```

Comment:
`hasReachedEnd` would be the safe fallback, as `isChatEndReached.value` might be incorrect for e.g. federated conversations, where we assume last message id and do not know it for sure

```suggestion
	function tryChatRelay(hasReachedEnd = false) {
		if (chatRelaySupported && (hasReachedEnd || isChatEndReached.value)) {
```